.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
this is just a tiny nit, but as this depends on your ide choice, this seems more a `~/.gitignore_global` entry rather than a project-level `.gitignore` entry